test/failed-directories.test.js

```diff
@@ -330,3 +330,12 @@ test.serial('with rcExtensions, find invalid JS in .foorc.js', (assert) => {
     assert.is(error.name, 'SyntaxError', 'threw correct error type');
   });
 });
+
+test.serial('Configuration file not exist', (assert) => {
```

Comment:
This new test is fine. Just to confirm: Does it fail on Windows without the above changes? It passes fine on my Mac, but that's to be expected.